dew/data/cmake/dew.cmake

```diff
@@ -7,7 +7,8 @@
 # It is encouraged to check this file into your project's VCS. Doing so will allow your cmake project to easily
 # integrate with Dew.
 #
-cmake_minimum_required(VERSION 3.2)
+cmake_minimum_required(VERSION 3.3)
+cmake_policy(SET CMP0057 NEW) # IN_LIST operator
```

Comment:
What does setting this policy do? Was there different behavior for IN_LIST in the past?

Reply:
CMP0057 simply *enables* the IN_LIST operator. Without the new policy, it's a parsing error:
```
CMake Error at dew.cmake:77 (if):
  if given arguments:

    "/home/jacko/PrimeWorldEditor/.dew/prefix-debug" "IN_LIST" "CMAKE_PREFIX_PATH"
```